otelcollector/main/main.go

```diff
@@ -56,7 +56,7 @@ func main() {
 		shared.StartCommand("powershell", "-NoProfile", "-ExecutionPolicy", "Bypass", "-File", "C:\\opt\\scripts\\filesystemwatcher.ps1")
 	}
 
-	if ccpMetricsEnabled != "true" {
+	if ccpMetricsEnabled != "true" && osType == "linux" {
```

Comment:
i am assuming this has no impact on current release (both AKS windows & ARC linux) ?